cluster/aws/coreos/util.sh

```diff
@@ -51,6 +51,9 @@ function generate-minion-user-data() {
       KUBERNETES_CONTAINER_RUNTIME=$(yaml-quote ${CONTAINER_RUNTIME})
       RKT_VERSION=$(yaml-quote ${RKT_VERSION})
 EOF
+  if [[ -n "${EXTRA_MINION_USER_DATA}" ]]; then
```

Comment:
I think we're in YAML for CoreOS configuration.  I think remove this and just add a comment saying "not currently supported for CoreOS" to config-default/config-test & options.md